cfme/tests/test_appliance.py

```diff
@@ -83,6 +83,14 @@ def test_httpd_running(ssh_client):
     assert 'is running' in stdout
 
 
+@pytest.mark.meta(blockers=[1336742])
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.5')
+def test_journald_running(ssh_client):
+    """Verifies systemd-journald service is running on the appliance"""
+    stdout = ssh_client.run_command('systemctl status systemd-journald')[1]
```

Comment:
Fixed, but I will be honest - I hate comments like this because I'm already onto something else at the time. I'm trying to fix a pressing issue and my PR is being held hostage because of something I had no idea about and something that is NOT in any of the other tests in the module. This comes back to keeping the code-base consistent, so please - next time something cool like this is introduced, enforce such change to show up in the whole codebase. Thank you. CCing @psav